### ISSUE TITLE
Run Chrome Lighthouse audit and create subsequent issues

### ISSUE THREAD

**Issue by craigcarr** [craigcarr/CountLog#62]

**Comment by craigcarr** [comment]
Lighthouse finds two issues. (1) The initial page load is not super quick. This can be addressed with code splitting. (2) Buttons are not labeled so that screen readers can work properly. I don't think this app will be used by very many people with screen readers, so it is not that important to fix.
